script.js

```diff
@@ -0,0 +1,172 @@
+//  ***********************  data base  ********************
+const recipes = [
+  {
+    id: 1,
+    title: "Gløgg",
+    picture_url:
+      "https://upload.wikimedia.org/wikipedia/commons/thumb/e/e2/Gl%C3%B6gg_kastrull.JPG/800px-Gl%C3%B6gg_kastrull.JPG",
+    ingredients: [
+      { NAME: "Orange zest", AMOUNT: "0.5" },
+      { NAME: "Water", AMOUNT: "200 ml" },
+      { NAME: "Sugar", AMOUNT: "275 g" },
+      { NAME: "Whole cloves", AMOUNT: "5" },
+      { NAME: "Cinnamon sticks", AMOUNT: "2" },
+      { NAME: "Spice", AMOUNT: undefined },
```

Comment:
It would be nice to hide ingredients with an `undefined` quantity.
<img width="381" alt="image" src="https://github.com/user-attachments/assets/4879b504-b680-4665-b5d1-b1d653984dd9" />

I don't know if undefined spices in my kitchen 🤔